release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -327,7 +354,7 @@ In the table below, features are marked with the following statuses:
 |Ephemeral Storage Limit/Requests
 |TP
 |TP
-|
+|TP
```

Comment:
@jsafrane Is this feature still in Tech Preview? I'm not familiar with its origin. It seems to have started in this table with 4.1. In 3.11, we had this: https://docs.openshift.com/container-platform/3.11/release_notes/ocp_3_11_release_notes.html#ocp-311-local-ephemeral-storage
Not sure if they're related, or if we even need this entry in the tracker anymore?

Reply:
Yes, it's still beta upstream.